src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -24,6 +24,10 @@ public List<Integer> getLottoNumbers() {
         return lottoNumbers.stream().map(LottoNumber::getNumber).collect(Collectors.toList());
```

Comment:
```suggestion
        return lottoNumbers.stream()
                .map(LottoNumber::getNumber)
                .collect(Collectors.toList());
```

이렇게 줄바꿈을 해주면 보편적으로 더 읽기 쉬워져요.